bindings/rust/s2n-tls-tokio/tests/shutdown.rs

```diff
@@ -157,34 +157,44 @@ async fn shutdown_with_blinding() -> Result<(), Box<dyn std::error::Error>> {
     let (mut client, mut server) =
         common::run_negotiate(&client, client_stream, &server, server_stream).await?;
 
-    // Trigger a blinded error for the server.
+    // Attempt to shutdown the client. This will eventually fail because the
+    // server has not written the close_notify message yet, but it will at least
+    // write the close_notify message that the server needs.
+    // Because time is mocked for testing, this does not actually take 10 minutes.
+    // TODO: replace this with a half-close once the bindings support half-close.
+    _ = time::timeout(time::Duration::from_secs(600), client.shutdown()).await;
```

Comment:
Just so that things are in a known state, could we assert on the expected result here? Also why `600` seconds?

Reply:
> Also why 600 seconds?
Just arbitrary wild overkill. If sending one messages takes more than 10min, you either have deeper problems or your environment is so broken as to be useless :) The mocked time makes wild overkill cheap.